### PR TITLE
[DOCS]Update spark dependency version notes

### LINKED ISSUE #1626: The doc of spark and flink version is not correct
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

Seatunnel 2.10 not support Spark3.x， the doc of [Quick start] about the spark version should not be "Spark version >= 2.x.x" ,  should be Spark version >= 2.x.x and < 3.x or spark3.x is not supported current. 

### SeaTunnel Version

v2.10

### SeaTunnel Config

```conf
nothing
```

### Running Command

```shell
nothing
```

### Error Exception

```log
nothing
```

### Flink or Spark Version

v3.x

### Java or Scala Version

java8

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chenhu**
issue line : https://github.com/apache/incubator-seatunnel/issues/1626#issue-1186058516

…stallation.md and quick-start.md
this closes #1626 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/development/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/zh-CN/spark/installation.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ https://github.com/apache/incubator-seatunnel/releases
 
 ### 准备Spark
 
-`Seatunnel` 依赖`Spark`。安装`seatunnel`前,需要准备好`Spark`。请先 [下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x`。 在下载和解压后, 不需要任何配置就可以指定 `deploy-mode = local` 提交任务。 如果需要使用其他模式，如 `Standalone cluster`，`Yarn cluster` ，`Mesos cluster`, 请参考官方文档。
+`Seatunnel` 依赖`Spark`。安装`seatunnel`前,需要准备好`Spark`。请先 [下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x `，目前还不支持spark3.x。 在下载和解压后,
```
Comment (ruanwenjun) [inline docs/zh-CN/spark/installation.md]:
```suggestion
`SeaTunnel` 支持`Spark`引擎，如果希望在`Spark`中使用`SeaTunnel`，需要提前准备好`Spark`环境。[下载 Spark](https://spark.apache.org/downloads.html)，选择 `Spark 版本 >= 2.x.x `，目前还不支持spark3.x。在下载和解压后，
```

**Review by ruanwenjun — COMMENTED** [review]
Thanks for your patch, I add a comment.
1. I think it's better to change `SeaTunnel`  rely on `Spark` to `SeaTunnel` support `Spark`
2. Maybe we need to use Chinese symbol in Chinese doc.

Please take a look.

**Review by ruanwenjun — DISMISSED** [review]
LGTM

**Comment by CalvinKirs** [comment]
hi, can you update the english documentation?

File: docs/zh-CN/spark/installation.md [hunk 2]
```diff
@@ -14,7 +14,10 @@ https://github.com/apache/incubator-seatunnel/releases
 
 ### 准备Spark
 
-`Seatunnel` 依赖`Spark`。安装`seatunnel`前,需要准备好`Spark`。请先 [下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x`。 在下载和解压后, 不需要任何配置就可以指定 `deploy-mode = local` 提交任务。 如果需要使用其他模式，如 `Standalone cluster`，`Yarn cluster` ，`Mesos cluster`, 请参考官方文档。
+`Seatunnel` 支持`Spark`引擎。如果希望在`Spark`中使用`SeaTunnel`，需要提前准备好`Spark`环境。[下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x 
+`，目前还不支持spark3.x。 在下载和解压后, 
+不需要任何配置就可以指定 
+`deploy-mode = local` 提交任务。 如果需要使用其他模式，如 `Standalone cluster`，`Yarn cluster` ，`Mesos cluster`, 请参考官方文档。
```
Comment (CalvinKirs) [inline docs/zh-CN/spark/installation.md:20]: Also, please pay attention to your document format, no line breaks are required here.

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**